README.md

```diff
@@ -328,6 +328,8 @@ import antfu from '@antfu/eslint-config'
 export default antfu({
   typescript: {
     tsconfigPath: 'tsconfig.json',
+    // or if you have multiple tsconfigs
+    tsconfigPath: ['tsconfig.json', 'tsconfig.node.json'],
```

Comment:
Copypasted code should "just work", so this line should be commented

It supports globs so I'd recommend adding `"packages/*/tsconfig.json"` here to imply that
And possibly `"tsconfig.eslint.json"` to imply you can make one especially for linting